parameters.cpp

```diff
@@ -464,7 +458,6 @@ bool P::addParameters() {
                         "maxvdt maxdt_acceleration " + "maxrdt maxdt_translation " +
                         "populations_maxvdt populations_maxrdt " +
                         "populations_maxdt_acceleration populations_maxdt_translation " +
-                        "populations_maxdistributionfunction populations_mindistributionfunction " +
```

Comment:
Wait, this and the _vg_ nuked above should remain in this deprecated list! Also then these two lists can take an updated date (keep the format) that gets reprinted by the check cfg script.

Reply:
Deprecated means "use of this is possible but not approved". 

Those outputs are no longer supported in any form.

Reply:
All right. Just the dates then.